web/satellite/vuetify-poc/src/components/billing/BillingHistoryTab.vue

```diff
@@ -2,14 +2,13 @@
 // See LICENSE for copying information.
 
 <template>
-    <v-card variant="flat" :border="true" rounded="xlg">
+    <v-card :border="true" rounded="xlg">
```

Comment:
Should this table have the same styling as the team, buckets, files, and projects tables?

Reply:
It's the same, doesn't really change anything.